versioneer.py

```diff
@@ -1454,9 +1526,8 @@ def versions_from_file(filename):
     return json.loads(mo.group(1))
 
 
-def write_to_version_file(filename, versions):
+def write_to_version_file(filename: str, versions: Dict[str, Any]) -> None:
     """Write the given version number to the given _version.py file."""
```

Comment:
One-line sphinx docstring (missing the description of input parameters, errors raised, what's returned, etc.: `:param` `:return`)